yb_migrate/src/utils/diagnostics.go

```diff
@@ -0,0 +1,143 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/fs"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
+)
+
+//call-home json formats
+var (
+	jsonFilePath    string
+	sendReq         sendRequest
+	jsonBuf         []byte
+	SendDiagnostics bool
+)
+
+type sendRequest struct {
+	MigrationUuid         uuid.UUID `json:"UUID"`                    //done
```

Comment:
since this struct is private, no use of making the struct members public. 
convert them to private by changing: `MigrationUuid` to `migrationUuid`

Reply:
Cannot do this, members of struct need to start with capital letters in order to be exported as json: https://stackoverflow.com/questions/50319404/has-json-tag-but-not-exported